hnn_core/network.py

```diff
@@ -1142,6 +1143,11 @@ def add_tonic_bias(self, *, cell_type=None, amplitude, t0=0, tstop=None):
             The name of the cell type to add a tonic input. When supplied,
             a float value must be provided with the `amplitude` keyword.
             Valid inputs are those listed in  `net.cell_types`.
+        section : str | 'soma'
```

Comment:
```suggestion
        section : str
```

The first line is only data type. The defaults are visible in the function signature or can be described in the line below